guides/training_keras_models_on_cloud.py

```diff
@@ -202,9 +202,9 @@ def create_model():
     callbacks = callbacks
     batch_size = 128
 else:
-    epochs = 1
-    callbacks = None
-    batch_size = None
+    epochs = 10
+    batch_size = 64
+    callback = None
```

Comment:
This should definitely be `callbacks`

Also, 1 epoch should be enough to debug

Reply:
Done.

Reply:
Keeping epochs at 5 so that the output on tf.org doesn't look sad